packages/core/admin/ee/server/content-types/workflow-stage/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = {
+  schema: {
+    collectionName: 'strapi_workflows_stages',
```

Comment:
Should this be 
```suggestion
    collectionName: 'strapi_workflow_stages',
```

?

Reply:
This collection will store all the stages for all the workflows, shouldn't we keep it plural ?